src/lib/prepareOas.ts

```diff
@@ -213,7 +209,9 @@ export default async function prepareOas(
   const specVersion: string = api.info.version;
   debug(`version in spec: ${specVersion}`);
 
-  if (['openapi', 'openapi inspect', 'openapi reduce'].includes(command)) {
+  const commandsThatBundle: (typeof command)[] = ['openapi inspect', 'openapi reduce', 'openapi upload'];
```

Comment:
🧐 do we want to bundle by default for `openapi upload`? see https://github.com/readmeio/rdme/issues/1142

Reply:
i think we have to because if we don't then they might upload an invalid spec that has ref pointers to a local file